src/components/Input/Input.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import InputTypes from './InputTypes';
+import './Input.scss';
+
+function Input(props) {
+  const { label, id, type, value, onValueChange, placeholder, errorString } = props;
+  return (
+    <div className={errorString !== "" ? "input__container input__container--error" : "input__container"}>
+      <label className="input__label" htmlFor={id}>
+        {label}
+      </label>
+      <input
+        id={id}
+        className="input"
+        type={type}
+        value={value}
+        placeholder={placeholder}
+        onChange={e => {
```

Comment:
Should we prefer to not have arrow function? Same as above similar review comment.

Reply:
Does not apply here.